edu.harvard.i2b2.crc/src/server/edu/harvard/i2b2/crc/dao/setfinder/querybuilder/ValueConstrainsHandler.java

```diff
@@ -128,8 +128,8 @@ public void setUnitCdConversionFlag(boolean unitCdConverstionFlag,
 				constraintSql += " valtype_cd = 'B' AND ";
 				// constrainSql = " valtype_cd = 'B' AND " ;
 				if (oracleFlag == true) {
-					constraintSql += " instr(observation_blob,'"
-							+ containsSql + "') ";
+					constraintSql += " regexp_instr(observation_blob,'"
+							+ containsSql + "' ,1 , 1, 0, 'i') ";  // replace instr with regexp_instr for case-insensitive matching
```

Comment:
usually "replace ..." only goes in the change comment.

But a comment to explain that 'i' means case-insensitive is worthwhile, so I don't mind leaving this here.